SUPPORT_POLICY.rst

```diff
@@ -22,16 +22,17 @@ This table describes the current support status of each major version of the AWS
       - Next status
       - Next status date
     * - 1.x
-      - Maintenance
       - End of Support
-      - 2022-07-08
+      - 
+      - 
     * - 2.x
-      - Maintenance
       - End of Support
-      - 2022-07-15
+      - 
+      - 
     * - 3.x
       - Generally Available
-      -
+      - Maintenance
+      - 2023-07-23
       -
```

Comment:
4 Columns, 4 `-`. 
```suggestion
```

Reply:
Weird that it added an extra `-`. I just had accepted the suggestion without making any change on my side: https://github.com/aws/aws-dynamodb-encryption-python/pull/518#discussion_r1073097544

Reply:
Yes, there are limits to what I can "suggest".
The last `-` was too far from the changed content,
so GitHub would not allow me to suggest a change to it. 

But you can always ask GitHub to render the Markdown from your branch,
and then you will know if your table is formatted correctly.